src/v1/steward.ts

```diff
@@ -45,6 +46,7 @@ export function handleBuy(event: Buy): void {
   let tokenIdString = tokenIdBigInt.toString();
   let ownerString = owner.toHexString();
   let txHashString = event.transaction.hash.toHexString();
+  let currentTimestamp = event.block.timestamp;
```

Comment:
I think no need to change it in this PR, but some places say `currentTimestamp` others say `txTimestamp`. We should make this consistent.

Reply:
Yes I saw this was different to your PR.